Readability.js

```diff
@@ -709,6 +708,13 @@ Readability.prototype = {
     this._cleanConditionally(articleContent, "ul");
     this._cleanConditionally(articleContent, "div");
 
+    //scale down h2-h5 because it's too large most of the time (intro's in h2, etc)
+    this._replaceNodeTags(this._getAllNodesWithTag(articleContent, ["h5"]), "h6");
+    this._replaceNodeTags(this._getAllNodesWithTag(articleContent, ["h4"]), "h5");
+    this._replaceNodeTags(this._getAllNodesWithTag(articleContent, ["h3"]), "h4");
+    this._replaceNodeTags(this._getAllNodesWithTag(articleContent, ["h2"]), "h3");
```

Comment:
The comment doesn't seem like a great reason to change the semantics of the headers - the "right" solution would probably to revert intro paragraphs into, well, paragraphs, instead of headers, if they meet some threshold / algorithm.